.github/workflows/backend-tests.yml

```diff
@@ -70,8 +70,45 @@ jobs:
         working-directory: .
         run: ./scripts/test.backend.sh
 
+  format:
```

Comment:
`format` was moved to `backend-checks.yml`

Reply:
done